specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -10468,6 +10476,13 @@
         "useManagedDisks": {
           "description": "A value indicating whether managed disks should be used during failover.",
           "type": "string"
+        },
+        "diskIdToDiskEncryptionMap": {
+          "description": "The disk Id to disk encryption set map.",
```

Comment:
Again, please provide additional description around what "disk ID" is and what its looking for in the value (disk encryption set RESOURCE ids)